src/map/mob.cpp

```diff
@@ -1263,6 +1263,24 @@ static int32 mob_can_changetarget(struct mob_data* md, struct block_list* target
 	}
 }
 
+/**
+ * Randomizes the target ID of a monster if it has the given mode
+ * @param bl Unit that is going to attack
+ * @param target_id Target ID to modify
+ */
+void mob_randomtarget(mob_data& md, int32& target_id) {
+	if (!(md.status.mode&MD_RANDOMTARGET))
```

Comment:
```suggestion
	if (!status_has_mode(md.status, MD_RANDOMTARGET))
```

Reply:
Here we already have md. I thought status_has_mode is only needed when the bl type is unknown?